README.md

```diff
@@ -602,10 +602,11 @@ Request body example:
 Running the DELETE request will remove the preferred_terminology collection
 from the `Terminology` specified by the id
 
-### https://[APPURL]/api/Terminology/[id]/user_input/[code]/[type]
+### https://[APPURL]/api/Terminology/[id]/user_input/[code]/[mapped_code]/[type]
```

Comment:
Let's have a constant between [code] and [mapped_code] to clarify the purpose of the relationship. I suggest [code]/mapping/[mapped_code] or something along those lines.